scenarios/import_td_logs/README.md

```diff
@@ -0,0 +1,36 @@
+# Workflow: Import Treasure Data Logs from Data Landing Area
+This example shows how you can use workflow to ingest Treasure Data Logs From Data Landing Areas to your Treasure Data account. 
+
+# How to Run
+## Requirement
+The workflow requires that Data Landing Areas feature is enabled in your Treasure Data account and thne you've got your User ID to access to it.
+
+## Steps
+First, edit configurations. You can find the following settings in the `import_td_logs.dig` file.
+
+|  Parameter |  Description  |
+| ---- | ---- |
+|  api_endpoint  | The endpoint of the Treasure Data API. See this [document]('https://docs.treasuredata.com/display/public/PD/Sites+and+Endpoints'). (e.g. https://api.treasuredata.com) |
```

Comment:
This link markdown might not be unfolded.

```suggestion
|  api_endpoint  | The endpoint of the Treasure Data API. See this [document](https://docs.treasuredata.com/display/public/PD/Sites+and+Endpoints). (e.g. https://api.treasuredata.com) |
```

Reply:
Thanks! Will fix.